src/components/App.js

```diff
@@ -12,26 +13,36 @@ function App(props) {
         <Header />
 
         <div className="warning">
-          <p>
-            The safety of your funds is important to us.
-          </p>
+          <p>The safety of your funds is important to us.</p>
           <p>
             This dApp is in ALPHA and improper use may lead to LOSS OF FUNDS.
           </p>
           <p>
             For more information and options please{" "}
-            <a href="https://discord.gg/Bpzfsgx">visit our Discord community</a>.
+            <a href="https://discord.gg/Bpzfsgx">visit our Discord community</a>
+            .
           </p>
         </div>
 
-        { children }
+        {children}
       </div>
-      <Footer includeSubscription={
-        location.pathname === '/' ||
-        location.pathname.startsWith('/news') // TODO: remove when proper CMS is selected
-      } />
+      <Footer
+        includeSubscription={
+          location.pathname === "/" || location.pathname.startsWith("/news") // TODO: remove when proper CMS is selected
```

Comment:
Do we have a `/news` here, or should this simplify to just `location.pathname === "/"` now?